WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="form-group same-line">
+  Event: <br /<br /><%= select_tag(:event, options_for_select(@events, params[:event])) %>
```

Comment:
The html formatting looks broken here... `<br /<br />`

Can you include a screenshot of how this looks on mobile?

What does the `same-line` class do? I don't see that on the current website.